libconnman-qt/networkmanager.cpp

```diff
@@ -675,6 +675,18 @@ void NetworkManager::updateServices(const ConnmanObjectList &changed, const QLis
     for (const ConnmanObject &obj : changed) {
         const QString path(obj.objpath.path());
 
+        // Ignore all WiFi with a zeroed/unknown BSSIDs to reduce list size
+        // in crowded areas. These are most likely weak and really unreachable
+        // but ConnMan maintains them if they come within reach and then they
+        // have a valid BSSID. WiFi services with an empty BSSID are saved ones
+        // that are not in the range.
+        if (obj.properties.value("Type").toString() == NetworkManager::Private::WifiType) {
+            const QString bssid = obj.properties.value("BSSID").toString();
+
+            if (bssid == "00:00:00:00:00:00")
```

Comment:
Could use QStringLiteral for the latter.

Sort of assumes bssid never changes into 00 etc, but guess that's good.

Reply:
AFAIK the network is gone in that situation if the BSSID goes to zeros.

Reply:
And anyways should not happen this way in ConnMan, if it was earlier visible we keep the BSSID.

Reply:
QStringLiteral added with 4595ac75213aed8411d381e689c96fecda0500b6